dashboard/app/models/foorm/form.rb

```diff
@@ -43,13 +43,32 @@ def self.setup
 
     transaction do
       reset_db
-      Foorm::Form.import! foorms
+      Foorm::Form.import! forms
     end
   end
 
-  def self.get_form_and_latest_version_for_name(form_name)
+  def self.get_questions_and_latest_version_for_name(form_name)
     latest_version = Foorm::Form.where(name: form_name).maximum(:version)
     form = Foorm::Form.where(name: form_name, version: latest_version).first
-    return form, latest_version
+
+    # Substitute any questions from the library.
+    questions = JSON.parse(form.questions)
```

Comment:
something to call out--this currently won't substitute library items in a panel (a panel is a type of element that contains elements).

Reply:
also I noticed yesterday panels can be nested!